source/components/n2o_component.cpp

```diff
@@ -96,19 +96,19 @@ void N2OComponent::setData( const string& varName,
     try {
         if( varName == D_PREINDUSTRIAL_N2O ) {
             H_ASSERT( data.date == Core::undefinedIndex() , "date not allowed" );
-            N0 = data.getUnitval(U_PPBV_N2O);
+            N0 = data.getUnitval( U_PPBV_N2O );
         } else if( varName == D_EMISSIONS_N2O ) {
             H_ASSERT( data.date != Core::undefinedIndex(), "date required" );
-            N2O_emissions.set(data.date, data.getUnitval(U_TG_N2O));
+            N2O_emissions.set(data.date, data.getUnitval( U_TG_N ) );
         } else if( varName == D_NAT_EMISSIONS_N2O ) {
             H_ASSERT( data.date != Core::undefinedIndex(), "date required" );
-            N2ON_emissions.set(data.date, data.getUnitval(U_TG_N2O));
-        } else if( varName == D_CONVERSION_N2O ) {
+            N2O_natural_emissions.set(data.date, data.getUnitval( U_TG_N ) );
+        }  else if( varName == D_CONVERSION_N2O ) {
             H_ASSERT( data.date == Core::undefinedIndex(), "date not allowed" );
             UC_N2O = data.getUnitval(U_TG_PPBV);
         } else if( varName == D_INITIAL_LIFETIME_N2O ) {
             H_ASSERT( data.date == Core::undefinedIndex(), "date not allowed" );
-            TN2O0 = data.getUnitval(U_YRS);
+            TN2O0 = data.getUnitval( U_YRS );
```

Comment:
I was just trying for consistency. Most of the code follows the GCAM convention (which I don't like...).

Reply:
> A foolish consistency is the hobgoblin of little minds, adored by little statesmen and philosophers and divines. With consistency a great soul has simply nothing to do. He may as well concern himself with his shadow on the wall.
   --Ralph Waldo Emerson, _Self Reliance_

> Consistency is contrary to nature, contrary to life. The only completely consistent people are the dead.
  -- Aldous Huxley, _Do What You Will_

> Consistency is the last refuge of the unimaginative.
  -- Oscar Wilde, "The Relation of Dress to Art"

> Consistency is contrary to nature, contrary to life. The only completely consistent people are the dead.
  -- Aldous Huxley, _Do What You Will_

> Consistency is the last refuge of the unimaginative.
  -- George Bernard Shaw, quoted in _Bernard Shaw: The Lure of Fantasy_

Who are we to argue with those luminaries?  

(To be clear, I'm not saying you should go back and change it; I just don't see any reason for you (or me) to write code that we consider to be ugly just because someone else did.)